arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -1151,1135 +1149,5009 @@
+          },
+          {
+            "name": "integrationAccount",
+            "description": "The integration account.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IntegrationAccount"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/IntegrationAccount"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "IntegrationAccounts"
+        ],
+        "operationId": "IntegrationAccounts_Delete",
+        "description": "Deletes an integration account.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "The resource group name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "integrationAccountName",
+            "description": "The integration account name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "No Content"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/integrationAccounts/{integrationAccountName}/listCallbackUrl": {
+      "post": {
+        "tags": [
+          "IntegrationAccounts"
+        ],
+        "operationId": "IntegrationAccounts_ListCallbackUrl",
+        "description": "Lists the integration account callback URL.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "The resource group name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "integrationAccountName",
+            "description": "The integration account name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "name": "parameters",
+            "description": "The callback URL parameters.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ListCallbackUrlParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CallbackUrl"
```

Comment:
The name of the operation is List where as the return type of the object is a string. The customer would expect that the service would return an array or an object that has some property that is of type array. Please fix the name of the method to _GetCallbackUrl